src/main/java/org/gridsuite/voltageinit/server/service/parameters/VoltageInitParametersService.java

```diff
@@ -129,27 +143,40 @@ private void setLowVoltageLimit(List<VoltageLimitOverride> specificVoltageLimits
             } else {
                 newLowVoltageLimit = voltageLimit;
             }
-            specificVoltageLimits.add(
-                new VoltageLimitOverride(voltageLevel.getId(),
-                    VoltageLimitOverride.VoltageLimitType.LOW_VOLTAGE_LIMIT,
-                    false,
-                    newLowVoltageLimit
-                ));
+            specificVoltageLimits.add(new VoltageLimitOverride(voltageLevel.getId(), VoltageLimitType.LOW_VOLTAGE_LIMIT, false, newLowVoltageLimit));
+            counterMissingVoltageLimits.increment();
```

Comment:
With that code, a missing voltage replacement will be counted one time for low limit, one time for high limit. We don't want that

Reply:
Same for counterVoltageLimitModifications